Resources/doc/suggesters/types.rst

```diff
@@ -0,0 +1,89 @@
+Suggester Types
+===============
+
+There are four basic types of suggesters objects to be used in a query and they have different parameters:
```

Comment:
There are four basic types of suggester objects to be used in a query and they have different parameters: